package.json

```diff
@@ -3,12 +3,15 @@
   "version": "0.0.1",
   "description": "A starting point for the CSS Frameworks CA project",
   "scripts": {
-    "test": "echo \"We will learn more about testing in the Workflow course\" && exit 1"
-  },
+    "build": "sass src/scss:dist",
```

Comment:
Do you mean `sass src/scss:dist/css`

Reply:
Also, the package.json is wrongly formatted and that is why vscode highlight with red color.  See the correct format below
```
{
  "name": "css-frameworks-ca",
  "version": "0.0.1",
  "description": "A starting point for the CSS Frameworks CA project",
  "scripts": {
    "build": "sass src/scss:dist",
    "watch": "sass --watch src/scss:dist/css"
    },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/AnaCascante/2023-09-03_CSS_Frameworks_CA_Ana-Cascante_Repo.git"
  },
  "keywords": [
    "CSS",
    "CSS",
    "Frameworks",
    "Bootstrap",
    "SCSS",
    "SASS"
  ],
  "author": "Ana Cascante <anacas71213@stud.noroff.no>",
  "license": "ISC",
  "bugs": {
    "url": "https://github.com/AnaCascante/2023-09-03_CSS_Frameworks_CA_Ana-Cascante_Repo.git/issues"
  },
  "homepage": "https://github.com/AnaCascante/2023-09-03_CSS_Frameworks_CA_Ana-Cascante_Repo.git#readme",
  "main": "index.js",
  "dependencies": {
    "bootstrap": "^5.3.1"
  },
  "devDependencies": {
    "sass": "^1.66.1"
  }
}

```

Reply:
I corrected  "sass src/scss:dist" I can see that I wrote it correctly in "watch" but missing the css in "build".

Reply:
I can see the mistake that I made in the package.json, I did not put all the elements inside the same {}- now this is fixed.